cmd/restore.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
```

Comment:
As this file is entirely new, I know you wrote it in 2017, however it is likely to get merged in 2018 should this date be updated to 2018?

Reply:
The copyright year should be the time first created I think.